tests/triggers/kit/small/upstream-load.yaml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
This seems like there will be a lot of duplicate test code for EKS and KIT clusters. Can we keep using the same test specs and parameterize required fields like endpoints, S3 buckets?